docs/design/3_1_2_platform_management.tex

```diff
@@ -11,14 +11,16 @@ \subsubsection{Subsystem Overview}
 \end{figure}
 
 \subsubsection{Device Selection}
-The selected device to implement the platform management system is the Raspberry Pi 4. As seen in Figure \ref{rpi}, the Raspberry Pi 4 is a small single-board computer which hosts an ARM Cortex A72 CPU, a CSI camera interface, 40 serial pins, WiFi/Bluetooth, and a gigabit Ethernet port. The Raspberry Pi 4 was selected over other single-computer platforms (such as Arduino/Beagleboard) as it:
+The selected device to implement the platform management system is the Raspberry Pi 4. As seen in Figure \ref{rpi}, the Raspberry Pi 4 is a small single-board computer which hosts a quad-core ARM Cortex A72 CPU, a CSI camera interface, 40 serial pins, WiFi/Bluetooth, and a gigabit Ethernet port. The Raspberry Pi 4 was selected over other single-computer platforms (including Arduino/Beagleboard/Orange Pi) as it:
 \begin{itemize}
 \item is relatively inexpensive (CAN\$45 in November 2019),
 \item has a large support community and a wealth of driver/library support,
 \item supports a wide range of high throughput I/O standards (USB/WiFi/Ethernet/Serial, \textbf{C.EX.3}),
-\item provides an industry-standard camera interface (\textbf{C.EX.5}), and
-\item supports extensible non-volatile storage (\textbf{F.CP.4}).
-\item is regularly succeeded with cross-compatible revisions (such as a Raspberry Pi 5), future-proofing the PMB segment
+\item provides an industry-standard camera interface (\textbf{C.EX.5}),
+\item supports extensible non-volatile storage (\textbf{F.CP.4}),
+\item has been regularly succeeded with cross-compatible revisions (presumably, including the upcoming Raspberry Pi 5) -- future-proofing the PMB segment,
```

Comment:
```suggestion
\item has been regularly succeeded with cross-compatible revisions (presumably, including the upcoming Raspberry Pi 5) --- therefore future-proofing the PMB segment,
```

I'm also unsure of the word-choice of "succeeded with". Do you mean that multiple revisions have surpassed the current version (successors) or do you mean that the platform is successful, so future revisions would most likely be compatible?

Reply:
The first one. That being said, I agree that's unclear, how about this:

```suggestion
\item is presumed to have a cross-compatible successor (the Raspberry Pi 5) -- future-proofing the PMB segment,
```

Reply:
That's better